packages/react-static/src/static/webpack/webpack.config.prod.js

```diff
@@ -165,18 +164,29 @@ export default function(state) {
   result.devtool = false
   result.externals = [
     new RegExp(`${paths.PLUGINS}`),
+    // Externalise all node_modules except for react-static files that need to be bundled
     (context, request, callback) => {
-      const resolved = path.resolve(context, request)
+      // All paths starting with `./` or `/` may be bundled as usual
+      if (/^[./\\]/.test(request)) {
+        return callback();
+      }
+
+      // Attempt to resolve the requested module or file from `context` which points at
+      // the file's location where the dependency on `request` is located at.
+      const res = resolveFrom(`${context}/`, request);
+
```

Comment:
Shouldn't this also include this loader check/webpack check: https://github.com/Timer/next.js/blob/31a47b4bffa45fc9d5f2bd464660bb68f6037229/packages/next/build/webpack-config.ts#L536-L542